frontend/src/components/eMIB/EmibIntroductionPage.jsx

```diff
@@ -11,32 +17,36 @@ const styles = {
 
 class EmibIntroductionPage extends Component {
   static propTypes = {
-    nextPage: PropTypes.func.isRequired
+    nextPage: PropTypes.func.isRequired,
+    // Provided by Redux
+    getTestMetaData: PropTypes.func
+  };
+
+  state = {
+    markdown_en: "",
+    markdown_fr: ""
+  };
+
+  // loads the markdown content (english and french versions)
+  componentWillMount = () => {
+    this.props.getTestMetaData(TEST_DEFINITION.emib.sampleTest).then(response => {
+      // saving the background information markdown content in local states
+      this.setState({
+        markdown_en: response.meta_text.en.overview[0],
+        markdown_fr: response.meta_text.fr.overview[0]
+      });
+    });
   };
 
   render() {
     return (
       <div>
-        <h1 className="green-divider">{LOCALIZE.emibTest.homePage.testTitle}</h1>
```

Comment:
We should keep the H1 here - we need an H1 on the page and we want it to be the title of the test (ideally this is loaded from the metadata API too).

Reply:
Fixed in a7e018d